includes/blocks/class-sensei-course-results-block.php

```diff
@@ -196,14 +222,22 @@ private function render_module( $item, $attributes ) {
 				]
 			);
 
-			$section_content[] = '<div ' . Sensei_Block_Helpers::render_style_attributes( 'wp-block-sensei-lms-course-results__module__separator', $header_border_css ) . '></div>';
+			$section_content[] = '<div ' . Sensei_Block_Helpers::render_style_attributes( 'wp-block-sensei-lms-course-results__separator', $header_border_css ) . '></div>';
 
 		}
 
+		if ( 0 < count( $item['lessons'] ) ) {
```

Comment:
Same here. Should it be one single `if`?

Reply:
Fixed in https://github.com/Automattic/sensei/pull/4279/commits/543facd07ef8077a0e28a56827564adfa1cacf4e.